test/soldiers.test.js

```diff
@@ -0,0 +1,87 @@
+import {
+  afterAll,
+  beforeAll, beforeEach, describe, expect, it,
+} from 'vitest';
+import { client } from '../connections.js';
+import app from '../app.js';
+import { dbName, dbCollection, addNewSoldier } from '../database.js';
+
+beforeAll(async () => {
+  await client.connect();
+});
+
+afterAll(async () => {
+  await client.close();
+});
+
+beforeEach(async () => {
+  await client.db(dbName).collection(dbCollection).deleteMany({});
+});
+
+describe('Get health route', () => {
+  it('checks that status is 200', async () => {
+    const res = await app.inject({
+      method: 'GET',
+      url: '/health',
```

Comment:
this is not soldier test?

Reply:
should I also have a health test file just for that one route?